tox.ini

```
@@ -42,106 +42,104 @@
 [tox]
 setupdir = {toxinidir}
 envlist =
-    python-adapter_cheroot-{py27,py36,py37,py38,py39,py310},
-    python-adapter_gevent-{py27,py36,py37,py38,py310},
-    python-adapter_gunicorn-{py36}-aiohttp1-gunicorn{19,latest},
-    python-adapter_gunicorn-{py36,py37,py38,py39,py310}-aiohttp3-gunicornlatest,
-    python-adapter_uvicorn-{py36,py37}-uvicorn03,
+    python-adapter_cheroot-{py27,py37,py38,py39,py310},
+    python-adapter_gevent-{py27,py37,py38,py310},
+    python-adapter_gunicorn-{py37,py38,py39,py310}-aiohttp3-gunicornlatest,
+    python-adapter_uvicorn-{py37}-uvicorn03,
```

```suggestion
    python-adapter_uvicorn-py37-uvicorn03,
```

The braces are for matrixing, where multiple versions can be specified and they're substituted in a matrix. When there's only 1 item left in a group they become unnecessary.